### PR TITLE
Reorganize getting started guides

### LINKED ISSUE #143: Update angular docs to include overlay in themes instructions
#### Describe the desired behavior
Theme instructions for angular should include adding the class to the overlay component as well otherwise dialogs etc. won't be themed.

#### Describe the current behavior
Instructions only mention the application root.

#### Is this request related to a current issue?
Pointed out by mackenzie:
image

### PR DESCRIPTION
**Pull request by joebochill**
Fixes #143 

Changes proposed in this Pull Request:
- Reorganize getting started guide contents
- Add information about the PX Blue CLI
- Add npm equivalents for yarn commands
- Fix error from prettier on the ImageGrid with two captions

### CODE REVIEW

File: src/docs/development/frameworks-mobile/cordova.mdx [hunk 1]
```diff
@@ -9,6 +9,8 @@ You'll need to start by following the general instructions for setting up your d
 The installation process for Cordova is a single command:
 
 ```
+npm install -g cordova
+or
```
Comment (huayunh) [inline src/docs/development/frameworks-mobile/cordova.mdx:13]: I'd put a // in front of all the "or"s.

```
npm install
// or
yarn add
```

File: src/docs/development/frameworks-mobile/react-native.mdx [hunk 2]
```diff
@@ -4,51 +4,62 @@ import { Divider } from '../../../app/components';
...
+<Divider />
+
+# Setting Up Your Environment
 
 You'll need to start by following the general instructions for setting up your development environment for mobile found in our [Environment Guide](/development/environment). In addition to the base requirements, you may also want the following:
 
 -   [Expo Client](https://expo.io/learn) (project creation / setup tool for React Native apps)(optional, for Expo projects)
 
-## Creating a project
+# Creating Your Application
```
Comment (huayunh) [inline src/docs/development/frameworks-mobile/react-native.mdx:15]: Keep consistent, use dividers in front of level one headlines

File: src/docs/development/frameworks-mobile/react-native.mdx [hunk 3]
```diff
@@ -57,19 +68,33 @@ yarn add @pxblue/react-native-components
...
+> Using the PX Blue React Native theme **requires** that you add the Open Sans font to your application. You can learn how to do this by reading the instructions for [Vanilla React Native](https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e) or [Expo](https://docs.expo.io/guides/using-custom-fonts/). You can find the fonts [here](https://github.com/pxblue/cli-helpers/tree/master/fonts).
+
 > When using Expo, you will need to specify the name for each font weight you load using the format OpenSans-`<Weight>`, e.g., OpenSans-SemiBold. Refer to one of our React Native demos for reference.
 
 ## Applying the Theme
 
-You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object. For detailed information, check [our documentation](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--theme).
+You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object.
+
+```
```
Comment (huayunh) [inline src/docs/development/frameworks-mobile/react-native.mdx:84]: \```tsx

File: src/docs/development/frameworks-mobile/cordova.mdx (hunk 1 above)
Comment (joebochill) [reply src/docs/development/frameworks-mobile/cordova.mdx]: good idea...done

File: src/docs/development/frameworks-mobile/react-native.mdx (hunk 2 above)
Comment (joebochill) [reply src/docs/development/frameworks-mobile/react-native.mdx:17]: Meant to do that and forgot...it's there now

File: src/docs/development/frameworks-mobile/react-native.mdx (hunk 3 above)
Comment (joebochill) [reply src/docs/development/frameworks-mobile/react-native.mdx]: I've added language type to all of the code blocks

**Comment by emclaug2** [comment]
There's an issue some dead links; is there a way to make the circle ci link check output more legible?

**Comment by jeffvg** [comment]
Github  throwing out Status: 429
HTTP 429 Too Many Requests response status code indicates the user has sent too many requests in a given amount of time ("rate limiting").

**Pull request merged**